chsdi/views/transports.py

```diff
@@ -0,0 +1,72 @@
+# -*- coding: utf-8 -*-
+
+from sqlalchemy import and_
+from pyramid.view import view_config, view_defaults
+from chsdi.models.vector.uvek import Oev_departures
+import datetime
+
+
+@view_defaults(renderer='jsonp', route_name='transports')
+class TransportView(object):
+
+    def __init__(self, request):
+        self.request = request
+        if request.matched_route.name == 'transports':
+            self.id = request.matchdict['id']
+
+    @view_config(request_method='GET')
+    def get_departures(self):
+        current_date = datetime.datetime.now()
+        #next_thirty_minutes = current_date + datetime.timedelta(minutes=30)
+        query = self.request.db.query(Oev_departures).filter(and_(Oev_departures.stop == self.id, Oev_departures.time > current_date)).order_by(Oev_departures.time).limit(10)
```

Comment:
It would be nice to have the limit returned results as a query parameter of the request. Same for transport types.
You can define default and upper values for these parameters.